bgpd/bgpd.c

```diff
@@ -3614,15 +3688,18 @@ static struct bgp *bgp_create(as_t *as, const char *name,
 	SET_FLAG(bgp->peer_self->cap, PEER_CAP_AS4_ADV);
 
 	bgp->peer = list_new();
+	bgp->peer_by_addr = list_new();
```

Comment:
ok I'm back to the question, why are we ever checking for the existence of this pointer?